README.md

```diff
@@ -1,73 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
 
-# Project Name
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-> Description the project.
+## Available Scripts
 
+In the project directory, you can run:
 
-## Built With
+### `npm start`
 
```

Comment:
- [ ] Kindly update the README using the Microverse README template and customize it to reflect what you have done in this project milestone so far.